apps/election-manager/src/components/StatusFooter.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useContext } from 'react'
+import styled from 'styled-components'
+import Text from './Text'
+import { localeWeedkayAndDate } from '../utils/IntlDateTimeFormats'
+import AppContext from '../contexts/AppContext'
+
+const StatusBar = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background: #455a64;
+  padding: 0.375rem 1rem;
+  color: #ffffff;
+`
+
+const StatusFooter: React.FC = () => {
+  const { electionDefinition } = useContext(AppContext)
+  if (electionDefinition === undefined) {
+    return null
+  }
+
+  const { election, electionHash } = electionDefinition
+  const electionDate =
+    election && localeWeedkayAndDate.format(new Date(election?.date))
```

Comment:
If `electionDefinition` is defined then `election` is too, right?
```suggestion
    localeWeedkayAndDate.format(new Date(election.date))
```